README.md

```diff
@@ -54,29 +54,50 @@ To use Secret Manager:
 pip install "prefect-gcp[secret_manager]"
 ```
 
-Then, register to [view the block](https://orion-docs.prefect.io/ui/blocks/) on Prefect Cloud:
+Then, register to [view the blocks](https://orion-docs.prefect.io/ui/blocks/) on Prefect Cloud:
 
 ```bash
 prefect block register -m prefect_gcp.credentials
+prefect block register -m prefect_gcp.cloud_run
 ```
 
 Note, to use the `load` method on Blocks, you must already have a block document [saved through code](https://orion-docs.prefect.io/concepts/blocks/#saving-blocks) or [saved through the UI](https://orion-docs.prefect.io/ui/blocks/).
 
-### Write and run a flow
+### Download blob from bucket
 
 ```python
 from prefect import flow
 from prefect_gcp import GcpCredentials
 from prefect_gcp.cloud_storage import cloud_storage_download_blob_as_bytes
 
-@flow()
-def example_cloud_storage_download_blob_flow():
+@flow
+def cloud_storage_download_blob_flow():
     gcp_credentials = GcpCredentials(
         service_account_file="/path/to/service/account/keyfile.json")
     contents = cloud_storage_download_blob_as_bytes("bucket", "blob", gcp_credentials)
     return contents
 
-example_cloud_storage_download_blob_flow()
+cloud_storage_download_blob_flow()
+```
+
+### Run command on Cloud Run
```

Comment:
This is a good example to have, but I think we need an example specifically for how we can use the `CloudRunJob` block in deployments. I think we could have a simple example here and link back to the core docs, but I'm open to other approaches.